wordpress-mobile/AztecEditor-iOS
# Removes EditContext from the code.

Fixes part of #492 

**To test:**

Simply run the app and make sure the editor works as before this change was applied.
